common/src/main/java/io/seata/common/loader/ExtensionURL.java

```diff
@@ -0,0 +1,101 @@
+/*
+ *  Copyright 1999-2019 Seata.io Group.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+package io.seata.common.loader;
+
+import io.seata.common.util.StringUtils;
+
+/**
+ * The type URL
+ *
+ * @author: haozhibei
+ */
+public class ExtensionURL {
```

Comment:
The class name is easy to misunderstand. What about ExtensionDefinition or ExtensionInfo.

Reply:
access left default done and how about ExtensionIdentifier?

Reply:
I think this class is not used to distinguish extensions, but describes extensions, may be include more attributes in the future, so ExtensionIdentifier (extension ID) is not very suitable. In my opinion, definition is more suitable.

Reply:
yeah,i agree with ServiceDefinition

Reply:
Yeah,i agree with ExtensionDefinition